libs/platform/src/lib/components/dynamic-page/dynamic-page.module.ts

```diff
@@ -35,7 +39,6 @@ import { DynamicPageComponent } from './dynamic-page.component';
         DynamicPageLayoutActionsComponent,
         DynamicPageHeaderComponent,
         DynamicPageContentComponent,
-        DynamicPageTabbedContentComponent,
         DynamicPageFooterComponent
     ]
 })
```

Comment:
can we also handle issue [5169](https://github.com/SAP/fundamental-ngx/issues/5169) here? Adding `DynamicPageService` in the `providers` array should be enough right?

Reply:
I think it's not what we need in this case. 
`DynamicPageService` must be dedicated (not singleton) for each instance of the dynamic page, otherwise we`ll get a singleton.